bigquery_etl/util/parallel_topological_sorter.py

```diff
@@ -42,7 +42,7 @@ def map(self, callback):
         processes = []
 
         if self.with_follow_up:
-            followup_queue = queue.Queue()
+            followup_queue = multiprocessing.Queue()
```

Comment:
this was the main issue, because `Queue` can't be shared safely across processes